cognite_toolkit/_cdf_tk/client/data_classes/charts_data.py

```diff
@@ -0,0 +1,303 @@
+import sys
+from dataclasses import dataclass, field, fields
```

Comment:
pydantic can be a lot nicer in my experience - and faster too - have you considered using it here? it might resolve the need to manually set camel_case everywhere...

Reply:
This follows the PySDK which we do not use pydantic as it is too big as I understand it to have as a dependency.